elastic/apm/track.json

```diff
@@ -10,7 +10,19 @@
   ],
   "component-templates": [
     {
-      "name": "apm@settings",
```

Comment:
I think you should change these to `traces@custom`, `metrics@custom`, and `logs@custom`. For example, the `metrics-apm.service_destination.1m-*` index template includes `metrics@custom` and `metrics-apm.service_destination.1m@custom`; it does _not_ include `metrics-apm@custom`.

Reply:
Thanks I updated the PR.